typescript/test/utils/mock-bridge.ts

```diff
@@ -314,4 +314,21 @@ export class MockBridge implements Bridge {
   walletRegistry(): Promise<WalletRegistry> {
     throw new Error("not implemented")
   }
+
+  wallets(walletPublicKeyHash: string): Promise<Wallet> {
+    throw new Error("not implemented")
+  }
+
+  buildUTXOHash(utxo: UnspentTransactionOutput): Hex {
+    return Hex.from(
+      utils.solidityKeccak256(
+        ["bytes32", "uint32", "uint64"],
+        [
+          utxo.transactionHash.reverse().toPrefixedString(),
+          utxo.outputIndex,
+          utxo.value,
+        ]
+      )
+    )
+  }
```

Comment:
In `typescript` dir they are only in 2 places in Ethereum implementation and in mock-bridge. However, I think it's hard to create a reusable function for that because it will only fit for cases where we want to hash 3 types and in exactly the same order. We can create the wrapper for `utils.solidityKeccak256` in `ethereum-helpers` but not sure if it's worth it.

Reply:
I see, let's keep it as it is now then